content/en/database_monitoring/setup/postgres/gcsql.md

```diff
@@ -0,0 +1,286 @@
+---
+title: Setting Up Database Monitoring for Google Cloud SQL managed Postgres
+kind: documentation
+description: Install and configure Database Monitoring for Postgres managed on Google Cloud SQL.
+code_lang: gcsql
+type: multi-code-lang
+code_lang_weight: 40
+further_reading:
+- link: "/tk/tk/"
+  tag: "Documentation"
+  text: "tktk"
+  
+---
+
+{{< site-region region="us3,gov" >}} 
+<div class="alert alert-warning">Database Monitoring is not supported in this region.</div>
+{{< /site-region >}}
+
+Database Monitoring provides deep visibility into your Postgres databases by exposing query metrics, query samples, explain plans, database states, failovers, and events.
+
+The Agent collects telemetry directly from the database by logging in as a read-only user. Do the following setup to enable Database Monitoring with your Postgres database:
+
+1. [Configure database parameters](#configure-postgres-settings)
+1. [Grant the Agent access to the database](#grant-the-agent-access)
+1. [Install the Agent](#install-the-agent)
+1. [Configure the Agent](#configure-the-agent)
+
+## Before you begin
+
+Supported PostgreSQL versions
+: 9.2, 10, 11, 12, 13
+
+Supported Agent versions
+: 7.30.0+
+
+Performance impact
+: The default Agent configuration for Database Monitoring is conservative, but you can adjust settings such as the collection interval and query sampling rate to better suit your needs. For most workloads, the Agent represents less than one percent of query execution time on the database and less than one percent of CPU. <br/><br/>
+Database Monitoring runs as an integration on top of the base Agent ([see benchmarks][1]).
+
+Proxies, load balancers, and connection poolers
+: The Agent must connect directly to the host being monitored. For self-hosted databases, `127.0.0.1` or the socket is preferred. The Agent should not connect to the database through a proxy, load balancer, or connection pooler. While this can be an anti-pattern for client applications, each Agent must have knowledge of the underlying hostname and should stick to a single host for its lifetime, even in cases of failover. If the Datadog Agent connects to different hosts while it is running, the values of metrics will be incorrect.
```

Comment:
```suggestion
: The Agent must connect directly to the host being monitored. For self-hosted databases, `127.0.0.1` or the socket is preferred. The Agent should not connect to the database through a proxy, load balancer, or connection pooler such as `pgbouncer`. While this can be an anti-pattern for client applications, each Agent must have knowledge of the underlying hostname and should stick to a single host for its lifetime, even in cases of failover. If the Datadog Agent connects to different hosts while it is running, the values of metrics will be incorrect.
```